dist/api_data.py

```diff
@@ -337,6 +337,10 @@ def __ge__(self, other):
         LSN of the last checkpoint'''),
     Config('id', '', r'''
         the file's ID number'''),
+    Config('live_restore', '', r'''
+        live restore metadata for a file''', type='category', subconfig=[
+            Config('bitmap', '', r'''bitmap representation of a file'''),
+            Config('bitmap_size', 0, r'''the size of the bitmap as an integer''', type='int')]),
```

Comment:
If this is the same as the number of bits allocated, I suggest using `nbits` to be consistent with the code in the bitstring file and incremental backup bitmap names.

Reply:
ba9df39c3